admission/Makefile

```diff
@@ -1,10 +1,10 @@
 CONTROLLER_TOOLS_VERSION = 0.14.0
-KUSTOMIZE_VERSION = 4.5.7
+KUSTOMIZE_VERSION = 5.2.1
 CONTOUR_VERSION = 1.28.2
-ARGOCD_VERSION = 2.10.1
+ARGOCD_VERSION = 2.10.2
 GRAFANA_OPERATOR_V4_VERSION = 4.10.1
 GRAFANA_OPERATOR_VERSION = 5.8.1
-ENVTEST_K8S_VERSION = 1.27.1
+ENVTEST_K8S_VERSION = 1.29.3
```

Comment:
'Regular update' steps do not instruct to update this line.
Could you explain why this is necessary?

Reply:
@morimoto-cybozu 
I made a mistake because there was an update on another product.
I will check the documentation more carefully.

Thanks a lot !